docs/index-operations.asciidoc

```diff
@@ -240,16 +240,16 @@ The Get Mappings API will return the mapping details about your indexes and type
 $response = $client->indices()->getMapping();
 
 // Get mappings for all types in 'my_index'
-$params = ['index' => 'my_index'];
+$params = [ 'index' => 'my_index'];
 $response = $client->indices()->getMapping($params);
 
 // Get mappings for all types of 'my_type', regardless of index
-$params = ['type' => 'my_type' ];
+$params = [ 'type' => 'my_type' ];
```

Comment:
That would not make the file consistent yet. Do you know the convention? I will update the file to match it.